<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionAndAnswerClass</class>
 <widget class="QMainWindow" name="QuestionAndAnswerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1262</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QuestionAndAnswer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="4">
     <widget class="QListWidget" name="listWidget"/>
    </item>
    <item row="0" column="4">
     <widget class="QListWidget" name="listWidget_2"/>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="addQbutton">
      <property name="text">
       <string>ADD Q</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="addAbutton">
      <property name="text">
       <string>ADD A</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="upVoteButton">
      <property name="text">
       <string>UP VOTE</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="downVoteButton">
      <property name="text">
       <string>DOWN VOTE</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1262</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QuestionAndAnswer.qrc"/>
 </resources>
 <connections/>
</ui>
